requirements.txt

```diff
@@ -0,0 +1,17 @@
+# =============================================================================
```

Comment:
If we're not actually using the `requirements.txt`, should we still have it in the repo?